SYCL/Basic/kernel_bundle/kernel_bundle_api.cpp

```diff
@@ -1,4 +1,3 @@
-// Disable fallback assert here until online-support is fixed.
```

Comment:
It might be worth asking @s-kanaev if this comment should be removed.  It looks to me like the test still has fallback assert disabled, so maybe the comment should remain until it is enabled.

Reply:
I'll ask about your concern.

Reply:
Fallback assert is now disabled by default. See https://github.com/intel/llvm/pull/4694.
Hence, this comment might be removed along with `-DSYCL_DISABLE_FALLBACK_ASSERT=1` in build line.
It'll be responsibility of the one who enables the fallback assert to verify it won't impact other tests.

P.S.: Sorry for late response.